components/org.wso2.carbon.identity.governance/src/main/java/org/wso2/carbon/identity/governance/listener/IdentityStoreEventListener.java

```diff
@@ -67,6 +69,12 @@ public IdentityStoreEventListener() throws IllegalAccessException, Instantiation
                 this.getClass().getName()).getProperties().get(DATA_STORE_PROPERTY_NAME).toString();
         Class clazz = Class.forName(storeClassName.trim());
         identityDataStore = (UserIdentityDataStore) clazz.newInstance();
+        Object hybridDataStoreEnableObject =
```

Comment:
We can cast to a String reference, and then pass to the parseBoolean function. No need to do null check also.

Reply:
Calling toString on the result of get() method without the null check might result in a NullPointerException if the get value is null right?

Reply:
The result of get() is an Object right?. we can cast it to String. Then we can pass that to the parse boolean function. We may not need null check and toString method

Reply:
Resolving due to these changes are related to backport changes from a public fix.